src/storage/mvcc/reader/reader.rs

```diff
@@ -256,15 +256,22 @@ impl<S: Snapshot> MvccReader<S> {
         }
     }
 
+    /// Gets `commit_ts` and `WriteType` of the transaction with specified `start_ts`.
+    /// The second return value indicates whether there is a write value with commit_ts equals to
+    /// the given `start_ts`.
     pub fn get_txn_commit_info(
         &mut self,
         key: &Key,
         start_ts: u64,
-    ) -> Result<Option<(u64, WriteType)>> {
+    ) -> Result<(Option<(u64, WriteType)>, bool)> {
```

Comment:
`The second return value indicates whether there is a write value with commit_ts equals to the given start_ts`
The return type is very strange, Option has indicated find or not.

Reply:
The option indicates whether there is a record in write cf whose start_ts is the same as the given `start_ts`, and the second value indicates whether there is a record whose commit_ts (the ts encoded in the key) is the same as the given `start_ts`.

Reply:
Too complicated.